server/fileupdater.mjs

```diff
@@ -325,4 +326,9 @@ function v7HolderClickable(properties) {
   if (properties.clickRoutine && !properties.clickable && properties.type=='holder'){
     properties.clickable=false;
   }
-}
\ No newline at end of file
+}
+
+function v8HoverInheritVisibleForSeat(properties) {
+  if (properties.onlyVisibleForSeat)
```

Comment:
I think this conditional needs to be removed - old files need this set to false on every widget to cancel the change of the default behavior that this PR introduces.

Reply:
This was discussed in meeting. I didn't understand it completely but it seems that only widgets with onlyVisibleToSeat would pass it to the hovering widget.

Reply:
That's right.